test/hotspot/gtest/classfile/test_symbolTable.cpp

```diff
@@ -37,7 +37,8 @@ TEST_VM(SymbolTable, temp_new_symbol) {
   Symbol* abc = SymbolTable::new_symbol("abc");
   int abccount = abc->refcount();
   TempNewSymbol ss = abc;
-  ASSERT_EQ(ss->refcount(), abccount) << "only one abc";
+  // TODO: properly account for Symbol cleanup delay queue
```

Comment:
I wonder if you can programmatically change the queue length to zero and keep these counts.

Then add a test with some loop of n being the queue length, and create n symbols and check that the first has been reclaimed?

Reply:
Good idea. I have implemented this in the next commit, but I'm not sure what the idiomatic way to expose this value for testing is - I have just done it naively so far.